template/.github/pull_request_template.md

```diff
@@ -5,14 +5,31 @@
 
 <!-- Commit message above. Everything below is not added to the message. Do not change this line! -->
 
-## Review Checklist
+## Definition of Done Checklist
 
+- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
+- Please make sure all these things are done and tick the boxes
+ 
+```[tasklist]
+# Author
+- [ ] Changes are OpenShift compatible
+- [ ] CRD changes approved
+- [ ] Helm chart can be installed and deployed operator works
```

Comment:
What about `For non-trivial PRs: Integration tests passed`?

Reply:
added (for the author) vs passed (for the reviewer) you mean?

Reply:
I meant passed. Normally the author starts the tests, so I'm fine with him checking the mark when the tests pass

Reply:
Just so I get it right: We already have "Integation tests added" for the reviewer.
Do you suggest to _move_ that to _Author_ and rename it to "passed" instead of added or do you suggest to have both?

Reply:
I would go with both